### PR TITLE
fixes #12 - add verbose option

### LINKED ISSUE #12: verbose option
could be nice with a verbose option that printed out:
```
- checking file /var/tmp/xxx.js
- checking file /var/tmp/xxx2.js
```

### PR DESCRIPTION
**Pull request by gemal**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,27 +52,28 @@ function resolveEditorConfig (e) {
 
 
 program.version(version)
-  .option('-n, --newline', 'Require newline at end of file.')
```
Comment (evanshortiss) [inline index.js]: I don't think there's a necessity for all the changes to these options lines. Can you undo this and simply leave it at the new verbose option you are adding?

File: index.js [hunk 2]
```diff
@@ -103,8 +105,15 @@ targetFiles = process.argv.slice(2).filter(fs.existsSync.bind(fs));
 
 
 // Run validation
+if (program.verbose) {
+  console.info("Number of files to check: " + targetFiles.length);
```
Comment (evanshortiss) [inline index.js]: Can you use single quotes to retain consistency with the existing code?

**Comment by evanshortiss** [comment]
Looks good @gemal, just some small changes requested and then I'll be happy to merge 👍

**Comment by gemal** [comment]
@evanshortiss how about now?

**Comment by evanshortiss** [comment]
@gemal merging. Will publish ASAP

**Comment by evanshortiss** [comment]
@gemal on npm as 0.4.0 - thanks for the PR 👍

**Pull request merged**